test/e2e/history_test.go

```diff
@@ -0,0 +1,46 @@
+package integration
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"path/filepath"
+)
+
+var _ = Describe("Podman export", func() {
```

Comment:
And this history_test looks to be the export test repeated?

Reply:
fixed